### ISSUE TITLE
tf.nn.embedding_lookup_sparse without combiner, to get result similar to tf.nn.embedded_lookup using dense input params

### ISSUE THREAD

**Issue by amadupu** [tensorflow/tensorflow#21482]
I would like to undrestand if `tf.nn.embedding_lookup_sparse` with sparse tensor can be used to get result similar to `tf.nn.embedded_lookup ` which requires dense representation of sparse tensor.

Currently im converting the sparse tensor to dense tensor (using `tf.sparse_tensor_to_dense`) and suppling the same as input to  `tf.nn.embedded_lookup `. This works well as expected

However from the naming convention it seemed `tf.nn.embedding_lookup_sparse` is doing the same process using the sparse tensor without the need to convert to dense representation. But the documentation specifies the method tries to combine the values in the dense representation based on some strategy. 

is there any way we can get the embeddng lookup on the sparse tensor, without combining the row values using the `tf.nn.embedding_lookup_sparse` 

One relavent question on stackoverlfow is copied for reference
[https://stackoverflow.com/questions/45789532/embedding-lookup-sparse-of-tensorflow-without-combiner](url)

**Comment by amadupu** [comment]
Have I written custom code: Yes

>                     dense = tf.sparse_tensor_to_dense(ws,name='dense')
>                     char_vec = tf.nn.embedding_lookup(embeddings, dense,name='embedded_lookup')
>                     # Output is Batch x  Max Words x Max Char x Char Feature Size

`vs`

>                     char_vec = tf.nn.embedding_lookup(embeddings, dense,name='embedded_lookup')
>                     # Output is Batch x  Max Words x Char Feature Size

I would like the output shape to be similar to the embedding lookup

`Batch x  Max Words x Max Char x Char Feature Size`






OS Platform and Distribution: Windows
TensorFlow installed from: pip install tensorflow
TensorFlow version: 1.9.0
Bazel version: Unknown
CUDA/cuDNN version: NA
GPU model and memory: NA
Exact command to reproduce: NA
Mobile device: NA

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rohan100jain: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rohan100jain: It has been 30 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rohan100jain: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rohan100jain: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rohan100jain** [comment]
Both of them use combiners to combine values, so I'm not sure what your question is here.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by facaiy** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
